### PR TITLE
Update README for better understanding

### PR DESCRIPTION
**Pull request by RoderickQiu**
Update README for better understanding. (Maybe it can help others~)

I think I'm going to make a theme here, but I don't exactly know when.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,16 +8,23 @@
...
-API Mock 调试服务器使用时，访问 `http://localhost:<PORT>/<THEME>/` 来浏览该主题的预览。
+API Mock 调试服务器启动后，可以访问 `http://localhost:<PORT>/<YOUR_THEME_NAME>/` 来浏览该主题的预览。
 
-需要事先运行 `THEME=<theme> yarn build` 来产生对应的 JS 和 CSS 才能工作。
+需要事先运行 `THEME=<YOUR_THEME_NAME> yarn build` 来产生对应的 JS 和 CSS 才能工作。
+
+需要说明的是，在 Windows 环境下，前置指定 `THEME` 环境变量无效，请在 PowerShell 中使用以下代码来替代：
+
+```powershell
+$env:THEME="<YOUR_THEME_NAME>" # 指定全局变量
```
Comment (moesoha) [inline README.md]: 指定环境变量

**Comment by moesoha** [comment]
Please follow the review comment to make a small change so that we can merge this pr.

**Comment by RoderickQiu** [comment]
Modified.

**Comment by moesoha** [comment]
Thank you.

**Pull request merged**